python/tvm/relay/prelude.py

```diff
@@ -591,6 +591,14 @@ def load_prelude(self):
         for global_def in GLOBAL_DEFS:
             setattr(self, global_def, self.mod.get_global_var(global_def))
 
-        for dtype in ['float32', 'int32']:
+        for dtype in ['float32',
```

Comment:
Is there a reason why we leave `uint32` and `uint64` out?

Reply:
There are issues executing `uin32` and `uint64` even inside TF itself. Looks like TF hasn't implemented the kernels for these 2 dtypes.

````python
            import tensorflow as tf            
            import numpy as np
            dtype_str = 'uint32'
            dtype = tf.uint32
            t = tf.constant(np.array([[1.0, 2.0], [3.0, 4.0]]).astype(dtype_str), dtype=dtype)
            t2 = tf.constant(np.array([[1.0, 2.0], [3.0, 4.0]]).astype(dtype_str), dtype=dtype)
            ta1 = tf.TensorArray(dtype=dtype, size=2, infer_shape=False, dynamic_size=False)
            ta2 = ta1.write(0, t)
            ta3 = ta2.write(1, t2)
            out = ta3.read(0)
            with tf.Session() as sess:
                sess.run(out)
````

Error is:
````
InvalidArgumentError: No OpKernel was registered to support Op 'TensorArrayWriteV3' used by node TensorArrayWrite/TensorArrayWriteV3 (defined at <ipython-input-5-05281037a3cf>:6) with these attrs: [T=DT_UINT32, _class=["loc:@Const"]]
Registered devices: [CPU]
Registered kernels:
  device='CPU'; T in [DT_INT64]
  device='CPU'; T in [DT_INT32]
  device='CPU'; T in [DT_UINT16]
  device='CPU'; T in [DT_INT16]
  device='CPU'; T in [DT_UINT8]
  device='CPU'; T in [DT_INT8]
  device='CPU'; T in [DT_HALF]
  device='CPU'; T in [DT_BFLOAT16]
  device='CPU'; T in [DT_FLOAT]
  device='CPU'; T in [DT_DOUBLE]
  device='CPU'; T in [DT_COMPLEX64]
  device='CPU'; T in [DT_COMPLEX128]
  device='CPU'; T in [DT_BOOL]
  device='CPU'; T in [DT_STRING]
  device='CPU'; T in [DT_RESOURCE]
  device='CPU'; T in [DT_VARIANT]
````

Reply:
I see. Thanks.